packages/i18nify-js/src/modules/phoneNumber/data/phoneRegexMapper.json

```diff
@@ -0,0 +1,253 @@
+{
+  "AF": "^(?:\\+93)?\\d{9}$",
+  "AX": "^\\+358\\d{5,8}$",
+  "AL": "^(?:\\+?355)?(?:[4-9]\\d{7}|6\\d{8})$",
+  "DZ": "^(?:\\+?213|0)([567]\\d{8})$",
+  "AS": "^(?:\\+1-684)?(?:\\d{7})$",
+  "AD": "^(?:\\+376)?\\d{6}$",
+  "AO": "^(?:\\+244)?(?:\\d{9})$",
+  "AI": "^(?:\\+1-264)?(?:\\d{7})$",
+  "AQ": "",
```

Comment:
Why is this empty ?

Reply:
Created this file from i18nify-data which must have this empty as well.